src/components/Radio/Radio.stories.tsx

```diff
@@ -0,0 +1,92 @@
+import { ComponentMeta, ComponentStory } from "@storybook/react";
+import React from "react";
+import { Radio } from "./Radio";
+
+export default {
+  title: "USWDS/Base/Radio",
+  component: Radio,
+  argTypes: {
+    id: {
+      description:
+        "A unique identifier for the input which must be unique in the whole document. Its purpose is to identify the element when linking. The value is used as the value of the `<label>`'s `for` attribute to link the label with the form control.",
+    },
+    isTile: {
+      description: "Use the tile variation of the Radio.",
+    },
+    label: {
+      description: "Label text that appears to the right of the Radio.",
+    },
+    name: {
+      description:
+        "Name of the input element. Radio components are grouped together by `name`.",
+    },
+    tileDescription: {
+      description:
+        "Text that can be used to describe the label in more detail. Appears only when `isTile` is set to `true`.",
+      if: { arg: "isTile" },
```

Comment:
Nice use of the conditional logic here.

Reply:
Per your suggestion on Slack, I've taken this out to show the `tileDescription` field all the time. Its conditional nature is already explained in the description, and I think you're 100% right, having it visible all the time may make it easier for other devs to understand. Great idea! 🎉

Reply:
I missed this. I didn't mean to take this out. If `tileDescription` is only associated with tile radios, then this is great. In slack I was linking to a comment about making `description` a feature of both radio types. If we're opposed to doing that, then having this conditional in Storybook makes sense.